OIPA/iati/models.py

```diff
@@ -597,6 +599,41 @@ class DocumentLinkTitle(models.Model):
         content_type_field='related_content_type',
         object_id_field='related_object_id')
 
+class DocumentSearch(models.Model):
+    document = models.OneToOneField('Document')
+    content = VectorField()
+    text = VectorField()
+    last_reindexed = models.DateTimeField()
+
+class Document(models.Model):
+    document_link = models.OneToOneField(DocumentLink)
```

Comment:
What are the pro's of OneToOne here? Couldn't we add the fields to the existing DocumentLink model?

The only problem I have with current functionality is that the API endpoint /document-links/ returns just the indexed document links and not all DocumentLink's reported in IATI. This will work confusing for someone that is used to IATI. 

Proposal; join the Document fields to DocumentLink, set field default to null where required. Make serializer work with DocumentLink model.